spec/refile/backend_examples.rb

```diff
@@ -215,5 +215,15 @@ def uploadable(data = "hello")
         expect(File.read(download.path)).to eq("hello")
       end
     end
+
+    describe "#as_json" do
+      it "returns id and stringified backend as a Hash" do
+        file = backend.upload(uploadable)
+        hash = file.as_json
+
+        expect(hash).to include(:backend)
+        expect(hash[:backend]).to be_a(String)
```

Comment:
Assert that only the keys id and backend are exposed, this will guarantee that no-one adds extra information unexpected to this method.

Something like:

```ruby
expect(hash.keys).to eq [:id, :backend]
```

Instead of the two assertions.

Reply:
change applied.